docs/serving/samples/cloudevents/cloudevents-go/index.md

```diff
@@ -5,4 +5,159 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
+A simple web app written in Go that can receive and send Cloud Events that you
```

Comment:
CloudEvents

Reply:
That's not really this PR, so you should check all the languages

Reply:
ok lgtm